tests/browser/pages/relying-party.js

```diff
@@ -31,6 +31,14 @@ module.exports = class PlaywrightDevPage {
     await this.page.goto(this.startingURL.toString());
   }
 
+  async gotoMultipleNames() {
+    if (process.env.USE_LOCAL_API === "false") {
+      this.startingURL = await this.getStartingURLForStubForMultipleNames();
```

Comment:
Is there a way we could do something with different users in config to do this rather than having to create a new function? Think we have something in address that could be helpful? https://github.com/govuk-one-login/ipv-cri-address-front/blob/ff8652c8ce871355dad7bb27042d4ff97d468c70/test/browser/support/env.js

Reply:
I've updated the tests to use the new `testUserConfig` file similar to the link you've pasted